### ISSUE TITLE
OVERRIDING keyword in postgres INSERT INTO statement is not parsable

### ISSUE THREAD

**Issue by markpolyak** [sqlfluff/sqlfluff#1858]
## Expected Behaviour
SQLFluff should be able to parse postgres `INSERT INTO` statements that are valid according to the [docs](https://www.postgresql.org/docs/current/sql-insert.html).

## Observed Behaviour
SQLFluff fails to parse valid postgres statement `INSERT INTO test (id, col1) OVERRIDING SYSTEM VALUE VALUES (1, 'val');`

## Steps to Reproduce
Run in `bash`:
```bash
echo "INSERT INTO test (id, col1) OVERRIDING SYSTEM VALUE VALUES (1, 'val');" | sqlfluff lint --dialect=postgres -
```

## Dialect
postgres

## Version
sqlfluff, version 0.8.1
Python 3.7.6

## Configuration
default

**Comment by jpy-git** [comment]
@tunetheweb I've got time to pick this one up too
